drivers/gles2/rasterizer_gles2.cpp

```diff
@@ -35,6 +35,7 @@
 #include "servers/visual/shader_language.h"
 #include "servers/visual/particle_system_sw.h"
 #include "gl_context/context_gl.h"
+#include "scene\main\scene_main_loop.h"
```

Comment:
DOS & Windows difer from UNIX/POSIX systems on a lot of things. Although I've been a DOS/Windows user for all my life (because that's the only thing I knew when started using computers) I've been disliking those useless "oddities" for some time now. Variety is good but only if it has a point.

Anyway, fixed and pushed.